designer/server/src/main/scala/pl/touk/nussknacker/ui/process/periodic/PeriodicProcessService.scala

```diff
@@ -296,6 +296,9 @@ class PeriodicProcessService(
         case (_, _, deployment, Some(status))
             if SimpleStateStatus.DefaultFollowingDeployStatuses.contains(status.status) =>
           deployment.id
+        // Without restarting, we silently succeed the restarting scenario cancel without sending anything to flink
```

Comment:
Maybe the comment could be a little bit more clear about the "restarting". Current version can be confusing, because "restarting" can be interpreted as "triggering/performing scenario restart", but it means "scenario in Restarting status" in this context.